lib/MetagenomeUtils/Utils/AMAUtils.py

```diff
@@ -40,3 +49,38 @@ def get_annotated_metagenome_assembly(self, params):
         })['data']
 
         return {'genomes': data}
+
+    def get_annotated_metagenome_assembly_features(self, params):
+        """
+        params: 
+            ref - workspace reference for KBaseMetagenomes.AnnotatedMetagenomeAssembly object
+        output:
+            features - list of features, each representing a dict.
+        """
+        ref = params['ref']
+        self._confirm_ws_type(ref)
+        ret = self.ws.get_objects2({
+            "objects": [{
+                "ref": ref,
+                "included": [
+                    "features_handle_ref"
+                ]
+            }]
+        })['data']
+        features_handle_ref = ret[0]['data']['features_handle_ref']
```

Comment:
I guess AnnotatedMetagenomeAssembly is guaranteed to have `features_handle_ref` field? Otherwise, it may raise KeyError.

Reply:
it should be guaranteed, but I can add a try except to catch that